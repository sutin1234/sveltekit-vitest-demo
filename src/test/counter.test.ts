import { cleanup, render } from '@testing-library/svelte'
import Counter from '../lib/Counter.svelte'
import { describe, it, expect, afterEach } from 'vitest'

describe('Counter.svelte', () => {
    afterEach(() => cleanup())

    it('Mounted', () => {
        const { container } = render(Counter)
        expect(container).toBeTruthy()
    })
})