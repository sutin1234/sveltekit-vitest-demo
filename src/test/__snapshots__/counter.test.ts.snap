// Vitest Snapshot v1

exports[`Hello.svelte > mountede Couter.svelte 1`] = `"<div><div class=\\"counter svelte-139m1ow\\"><button aria-label=\\"Decrease the counter by one\\" class=\\"svelte-139m1ow\\"><svg aria-hidden=\\"true\\" viewBox=\\"0 0 1 1\\" class=\\"svelte-139m1ow\\"><path d=\\"M0,0.5 L1,0.5\\" class=\\"svelte-139m1ow\\"></path></svg></button> <div class=\\"counter-viewport svelte-139m1ow\\"><div class=\\"counter-digits svelte-139m1ow\\" style=\\"transform: translate(0, 0%);\\"><strong style=\\"top: -100%;\\" aria-hidden=\\"true\\" class=\\"svelte-139m1ow\\">1</strong> <strong class=\\"svelte-139m1ow\\">0</strong></div></div> <button aria-label=\\"Increase the counter by one\\" class=\\"svelte-139m1ow\\"><svg aria-hidden=\\"true\\" viewBox=\\"0 0 1 1\\" class=\\"svelte-139m1ow\\"><path d=\\"M0,0.5 L1,0.5 M0.5,0 L0.5,1\\" class=\\"svelte-139m1ow\\"></path></svg></button></div></div>"`;
